### PR TITLE
Integration of calver

### PR DESCRIPTION
**Pull request by anuragstar**

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -18,6 +24,22 @@ const pad = (n) => {
...
+async function execCommand(command, options = {}) {
+  const projectPath = "/Users/anuragsinha/Documents/tools/mono-ios/hotstarx-ios-mobile"
+  options.cwd = projectPath
+  return exec.exec(command, [], options)
+}
+
+// function setValues(id, key) {
+//   for (var i = 0; i < jsonObj.length; i++) {
+//     if (jsonObj[i].Id === id) {
+//       jsonObj[i].keys = key;
```
Comment (rahulrathi-hotstar) [inline dist/index.js:36]: remove commented code

File: dist/index.js [hunk 2]
```diff
@@ -43,8 +65,9 @@ const isCalver = (version) => {
 // most @actions toolkit packages have async methods
 async function run() {
   try {
-    const filePath = core.getInput("path");
-    const platform = core.getInput("platform");
+    //const filePath = core.getInput("path");
+    const filePath = "/Users/anuragsinha/Documents/tools/mono-ios/hotstarx-ios-mobile/hotstarx-ios-mobile/Application/Info.plist"
```
Comment (rahulrathi-hotstar) [inline dist/index.js:69]: remove hardcoded filePath

File: dist/index.js.map [hunk 3]
```diff
@@ -1 +1 @@
...
...CA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;;;A;;;;;ACvQA;;;A;;;;;AAAA;;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;;;ACAA;;A;;;;ACAA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;;;AC7BA;AACA;;;;ACDA;AACA;AACA;AACA;;;;A","sourceRoot":""}
```
Comment (rahulrathi-hotstar) [inline dist/index.js.map:1]: projectPath is hardcoded

**Review by rahulrathi-hotstar — CHANGES_REQUESTED** [review]
changes needed

File: dist/index.js (hunk 1 above)
Comment (anuragstar) [reply dist/index.js:36]: Done

**Review by rahulrathi-hotstar — APPROVED** [review]

**Review by ankeetmaini — APPROVED** [review]

**Pull request merged**